vault-init-eso.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+enable_eso() {
+    local eso_service_account_namespace
+    local eso_service_account_name
+    local eso_pod_name
+
+    export SA_JWT_TOKEN=$(kubectl -n vault get secret auth-sa-token -o jsonpath="{.data.token}" | base64 --decode; echo)
```

Comment:
I can't find `auth-sa-token` secret in any namespace, and this chart creating only `auth-sa-eso-token` that will be used for External Secrets Operator auth, not kubernetes auth.

As mentioned [here](https://github.com/Alpacked/security-hardening-helm/pull/3#discussion_r1443206239), please ensure that secret is created or rework to use short-lived service account token.